terminus-ui/src/file-upload/file-upload.component.ts

```diff
@@ -319,13 +350,18 @@ export class TsFileUploadComponent implements OnInit , OnChanges, OnDestroy, Aft
     this._seedFile = file;
 
     if (file) {
-      const newFile = new TsSelectedFile(file, this.dimensionConstraints, this.acceptedTypes, this.maximumKilobytesPerFile);
+      const newFile = new TsSelectedFile(
+        file,
+        this.dimensionConstraints,
+        this.acceptedTypes,
+        this.maximumKilobytesPerFile,
+        this.ratioConstraints);
```

Comment:
Style: let's drop the closing `)` down a line and add a trailing comma. It can be very helpful for scanning when you see the indentation come back out to the original level.